### ISSUE TITLE
Installation problems

### ISSUE THREAD

**Issue by connecteev** [guillaumebriday/todolist-backend-laravel#9]
@guillaumebriday 
I've been trying to install this but it doesn't seem to work. 

I tried most of the installation steps here: https://github.com/guillaumebriday/todolist-backend-laravel#installation

After running ```php artisan serve``` and going to http://127.0.0.1:8000/ in the browser
I see
```
{
    "message": "",
    "exception": "Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException",
    "file": "/Users/kunalpunjabi/Code/vuejs_examples/guillaumebriday_todolist/todolist-backend-laravel/vendor/laravel/framework/src/Illuminate/Routing/RouteCollection.php",
    "line": 179,
    "trace": [
        {
            "file": "/Users/kunalpunjabi/Code/vuejs_examples/guillaumebriday_todolist/todolist-backend-laravel/vendor/laravel/framework/src/Illuminate/Routing/Router.php",
            "line": 634,
            "function": "match",
            "class": "Illuminate\\Routing\\RouteCollection",
            "type": "->"
        },
        {
            "file": "/Users/kunalpunjabi/Code/vuejs_examples/guillaumebriday_todolist/todolist-backend-laravel/vendor/laravel/framework/src/Illuminate/Routing/Router.php",
            "line": 623,
            "function": "findRoute",
            "class": "Illuminate\\Routing\\Router",
            "type": "->"
        },
        {
            "file": "/Users/kunalpunjabi/Code/vuejs_examples/guillaumebriday_todolist/todolist-backend-laravel/vendor/laravel/framework/src/Illuminate/Routing/Router.php",
            "line": 612,
            "function": "dispatchToRoute",
            "class": "Illuminate\\Routing\\Router",
            "type": "->"
        },

.
.
.
```

After running ```php artisan migrate``` I tried going through the steps here:
https://github.com/guillaumebriday/todolist-backend-laravel#accessing-the-api
and this is what I saw in the console:

```
$ curl -X POST http://localhost:8000/api/v1/auth/login -d "email=kp123@gmail.com&password=qwer1234"
{"errors":{"email":["These credentials do not match our records."]}}

$ curl -X POST http://localhost:8000/api/v1/auth/login -d "email=darthvader@deathstar.ds&password=4nak1n"
{"errors":{"email":["These credentials do not match our records."]}}

$ 

```

Note that php artisan route:list shows:
```
$ php artisan route:list
+--------+---------------+----------------------+---------------+--------------------------------------------------------------+------------------------------+
| Domain | Method        | URI                  | Name          | Action                                                       | Middleware                   |
+--------+---------------+----------------------+---------------+--------------------------------------------------------------+------------------------------+
|        | POST          | api/v1/auth/login    |               | App\Http\Controllers\V1\Auth\AuthController@login            | api,guest:api                |
|        | DELETE        | api/v1/auth/logout   |               | App\Http\Controllers\V1\Auth\AuthController@logout           | api,auth:api                 |
|        | GET|HEAD      | api/v1/auth/me       |               | App\Http\Controllers\V1\Auth\AuthController@me               | api,auth:api                 |
|        | POST          | api/v1/auth/refresh  |               | App\Http\Controllers\V1\Auth\AuthController@refresh          | api,auth:api                 |
|        | POST          | api/v1/auth/register |               | App\Http\Controllers\V1\Auth\RegisterController@register     | api,guest:api                |
|        | GET|HEAD      | api/v1/tasks         | tasks.index   | App\Http\Controllers\V1\TasksController@index                | api,auth:api                 |
|        | POST          | api/v1/tasks         | tasks.store   | App\Http\Controllers\V1\TasksController@store                | api,auth:api                 |
|        | DELETE        | api/v1/tasks         |               | App\Http\Controllers\V1\TasksController@deleteCompletedTasks | api,auth:api                 |
|        | GET|HEAD      | api/v1/tasks/{task}  | tasks.show    | App\Http\Controllers\V1\TasksController@show                 | api,auth:api                 |
|        | PUT|PATCH     | api/v1/tasks/{task}  | tasks.update  | App\Http\Controllers\V1\TasksController@update               | api,auth:api                 |
|        | DELETE        | api/v1/tasks/{task}  | tasks.destroy | App\Http\Controllers\V1\TasksController@destroy              | api,auth:api                 |
|        | PUT|PATCH     | api/v1/users/{user}  | users.update  | App\Http\Controllers\V1\UsersController@update               | api,auth:api,can:manage,user |
|        | DELETE        | api/v1/users/{user}  | users.destroy | App\Http\Controllers\V1\UsersController@destroy              | api,auth:api,can:manage,user |
|        | GET|POST|HEAD | broadcasting/auth    |               | Illuminate\Broadcasting\BroadcastController@authenticate     | auth:api                     |
+--------+---------------+----------------------+---------------+--------------------------------------------------------------+------------------------------+
```

What am I missing? As an aside, how do I even verify that this installation is working? :)
Btw I did go through the tutorial at https://guillaumebriday.fr/laravel-vuejs-faire-une-todo-list-partie-3-utiliser-les-api-resources but it didnt help.

**Comment by guillaumebriday** [comment]
You don’t need to run `php artisan serve` with docker, please stop this command if you want to avoid conflicts.

Your first response is not an error, please read the message. This project is an api, it’s not supposed to work in a browser.

Migration does not create an user for you. The credentials seem to not be valid, you need to register first or to use valid credentials.

If you follow the readme, it should be working fine.

Thank you @connecteev

**Comment by connecteev** [comment]
@guillaumebriday Thanks for the quick response Just trying to get this to work:

I am not using Docker and don't plan to (unless this code won't work without Docker, which would be a surprise).
You said ```The credentials seem to not be valid, you need to register first or to use valid credentials.``` How do I register?
How do I set the JWT? I keep getting "These credentials do not match our records.".

Btw I did run php artisan migrate --seed and it created an entry in the users table with 
```
Email : darthvader@deathstar.ds
Password : 4nak1n
```

The readme here: https://github.com/guillaumebriday/todolist-backend-laravel
has a ton of complicated steps, and I've tried most of them. It would be good to simplify the readme, but for now, I'm just trying to get the end to end demo to work. What am I doing wrong?
